internal/backends/winit/muda.rs

```diff
@@ -89,11 +89,46 @@ impl MudaAdapter {
             menu.init_for_nsapp();
         }
 
-        Self { entries: map, menubar, _menu: menu }
+        Self { entries: map, menubar: Some(menubar), _menu: menu }
     }
 
     pub fn invoke(&self, entry_id: usize) {
         let Some(entry) = &self.entries.get(entry_id) else { return };
-        self.menubar.activate(entry);
+        self.menubar.as_ref().unwrap().activate(entry);
```

Comment:
Is unwrap fine here? I guess because otherwise entries would be empty.

Reply:
I think it's fine, but OTOH might as well add a safeguard.